### ISSUE TITLE
[BUG] Html report showing wrong status

### ISSUE THREAD

**Issue by olivierodo** [restqa/cucumber-export#75]
On the html report even when the result of the test passed the report is showing that it is failing.

![image](https://user-images.githubusercontent.com/4768226/131824379-2c503f46-5b5d-45e6-8a75-fdbd02b4c40e.png)

### How to reproduce the bug.

1. clone the repository
2. install the dependencies `npm i`
3. run the command `npm run example:success`
4. click on the link to access to the html report

**Comment by olivierodo** [comment]
After some research the issue is coming from the migration to `cucumber@7`
The property `success` disappear from the original json response.
Current:
```json
  "id": "f26fbed2-8bb3-47cc-a638-fe3cb414c3b7",
  "startTime": "2021-09-02T08:10:02+00:00",
  "name": "Request query parameter recipe",
  "key": "REQUEST_QUERY_PARAMS",
  "env": "local",
  "duration": null,
  "durationFormat": "Invalid date",
  "timestamp": "2021-09-02T08:10:02+00:00",
  "type": "testSuite",
  "total": 1,
  "passed": 1,
  "failed": 0,
  "skipped": 0,
  "scenarios": {
    "passed": 2,
    "failed": 0,
    "skipped": 0,
    "undefined": 0
  },
```

The information should be added into here https://github.com/restqa/cucumber-export/blob/master/src/format.js#L55

We should loop each feature result and aggregate their result in order to get the test suite result.

**Comment by olivierodo** [comment]
This bug is also impacting the restqa dashboard.

![image](https://user-images.githubusercontent.com/4768226/131919180-9c01af65-ba81-4e85-a3c4-f565cf8da1c8.png)

**Comment by masonwoodford** [comment]
Hello, could I help with this?

**Comment by olivierodo** [comment]
Hi @masonwoodford, 
That would be great!

So basically at the root level of the `testSuite` object we should have the boolean property `success`.

**Comment by masonwoodford** [comment]
So is the idea that it is considered a success if all tests passed?

**Comment by olivierodo** [comment]
Exactly!

**Comment by masonwoodford** [comment]
I'm getting an error with the HTML report that there's no dist file in the html-report-template, is that part of the bug?

**Comment by olivierodo** [comment]
Oh no i forgot to add one step: to build the html report
```
1. clone the repository
2. install the dependencies `npm i`
3.Install dependencies for the html report : `npm run html-report:install`
4.Build the html report : `npm run html-report:build`
4. run the command `npm run example:success`
5. click on the link to access to the html report
```

**Comment by masonwoodford** [comment]
That fixed it, thanks!

**Comment by masonwoodford** [comment]
Just to clarify, a test with skipped or undefined scenarios would be considered passing as long as none are failing?

**Comment by olivierodo** [comment]
yes

**Comment by olivierodo** [comment]
Closing the issue we will integrate the fix on the [version 0.1.12](https://github.com/restqa/cucumber-export/pull/78).

Feel free to join our discord channel: https://restqa.io/chat we love to get your feedback on how we could improve our  contribution experience 😇.

**Comment by masonwoodford** [comment]
Awesome, thank you for the opportunity!
